openebs-hostpath.yaml

```diff
@@ -0,0 +1,153 @@
+# This manifest deploys the OpenEBS control plane components, with associated CRs & RBAC rules
+# NOTE: On GKE, deploy the openebs-operator.yaml in admin context
+
+# Create the OpenEBS namespace
+apiVersion: v1
+kind: Namespace
```

Comment:
@prateekpandey14 PTAL on this file

Reply:
file name is [hostpath-operator.yaml](https://github.com/openebs/charts/blob/gh-pages/hostpath-operator.yaml)  instead of `openebs-hostpath.yaml`

Reply:
Done.